.github/workflows/security-trivy-scan.yml

```diff
@@ -0,0 +1,116 @@
+name: Security - Scan Docker Image With Trivy
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+
+jobs:
+  security_scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+      - name: Build Docker image for security scan
+        id: build_image
+        run: |
+          IMAGE_TAG="n8n-security-scan:${{ github.sha }}"
+          docker build \
+            --tag $IMAGE_TAG \
+            --file ./docker/images/n8n/Dockerfile \
+            --build-arg N8N_RELEASE_TYPE=nightly \
+            .
+          echo "IMAGE_TAG=${IMAGE_TAG}" >> $GITHUB_OUTPUT
```

Comment:
instead of building another nightly image, we could add this as an additional job in [the existing nightly image building workflow](https://github.com/n8n-io/n8n/blob/master/.github/workflows/docker-images-nightly.yml).

Reply:
maybe the rest of the docker cleanup can happen in a separate PR, but for this I'd prefer if we could do this this way:
- remove `schedule` trigger from this workflow
- add an `image_tag` input to the `workflow_dispatch` trigger (which can default to `latest`)
- update `docker-images-nightly.yml` to add a new job that triggers this new workflow after `build` finishes, with `image_tag: nightly`
- update `release-publish.yml` to add a new job that triggers this new workflow after `publish-to-docker-hub`, with `image_tag: [[RELEASE_VERSION]]`